src/components/Txt/index.tsx

```diff
@@ -34,6 +34,14 @@ export const bold = (props: ThemedTxtProps) =>
 		  `
 		: null;
 
+export const truncate = (props: ThemedTxtProps) =>
+	props.truncate &&
```

Comment:
This will return `false` if truncate is false, and that will be added to the style as a string. We explicitly want to return an empty string here

Reply:
I was using `null` to keep the same pattern with those stated above.  This will not be added as false. 
ref:
```
Speaking of which, during flattening, styled-components ignores interpolations that evaluate to undefined, null, false, or an empty string (""), which means you're free to use short-circuit evaluation to conditionally add CSS rules.
```

https://styled-components.com/docs/advanced

luckily, the above rule applies, otherwise it would have been a problem even putting an empty string :)

Reply:
Empty string should have been fine, but I see SC handles it well :)